.babelrc.js

```diff
@@ -25,7 +25,10 @@ module.exports = {
   overrides: [
     {
       test: './examples/vue-kitchen-sink',
-      presets: [['@babel/preset-env', { shippedProposals: true }], 'babel-preset-vue'],
+      presets: [
+        ['@babel/preset-env', { shippedProposals: true, useBuiltins: 'usage' }],
```

Comment:
Since you use the same settings as on top level, you can omit this preset in the override

Reply:
Really? babel merges these?

Reply:
yep https://babeljs.io/docs/en/options#merging